src/input.rs

```diff
@@ -0,0 +1,49 @@
+use std::fmt;
+use crate::event::filter::{EventFilter};
+use crate::{csi, Command, Result};
+use crate::event::{Event, InternalEvent, read_internal};
+
+pub fn read() -> Result<Input> {
```

Comment:
In the past, we thought about why we call something an input or an event. We concluded that events are more general and include input, resize events and more. In this scenario, an input contains an event, but is the resize event an input?